proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/EidasAuthnRequestAppRuleTests.java

```diff
@@ -10,16 +10,19 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Captor;
 import org.mockito.Mock;
+import org.mockito.Spy;
```

Comment:
You've got some unused imports. IntelliJ is good at highlighting these and removing unnecessary ones (Code -> Optimise Imports or `alt+cmd+I`). Codacy picks that up as well.

Reply:
Yeah, I'm trying to commit the change but my yubikey is playing up. Nice intellij tip though, cheers!